src/Microsoft.Diagnostics.Monitoring.RestServer/MetricsService.cs

```diff
@@ -44,11 +46,23 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                         //TODO In multi-process scenarios, how do we decide which process to choose?
                         //One possibility is to enable metrics after a request to begin polling for metrics
                         IProcessInfo pi = await _services.GetProcessAsync(filter: null, stoppingToken);
-                        await _pipeProcessor.Process(pi.Client, pi.Pid, Timeout.InfiniteTimeSpan, stoppingToken);
+                        _counterPipeline = new EventCounterPipeline(pi.Client, new EventPipeCounterPipelineSettings
+                        {
+                            CounterGroups = Array.Empty<EventPipeCounterGroup>(),
```

Comment:
No counters are collected because there are no counter groups.

Reply:
We have defaults here that if there are no filters, we will use the default set of counter providers.